docs/multicluster/policy-only-mode.md

```diff
@@ -0,0 +1,131 @@
+Please check following steps to set up VPC Connectivity for a ClusterSet.
+
+Assuming there is a ClusterSet including two member clusters with following basic information:
+
+| Cluster ID   | ServiceCIDR   | PodCIDR       | Gateway IP   |
+| ------------ | ------------- | ------------- | ------------ |
+| west-cluster | 10.182.0.0/16 | 110.13.0.0/16 | 110.13.26.12 |
+| east-cluster | 10.183.0.0/16 | 110.14.0.0/16 | 110.14.18.50 |
+
+### VPC Peering Connection
+
+To set up VPC connectivity, you need to create VPC peering connection for member clusters.
+Following are steps about how to create VPC peering connection between two clusters. You may
+check latest [EKS document](https://docs.aws.amazon.com/vpc/latest/peering/what-is-vpc-peering.html)
+for the details.
+
+Go to `Virtual private cloud` -> `Peering connections`, click `Create peering connection`, set
+the name of peering connection, eg: `west-and-east` then configure a local VPC to peer with
+another VPC by selecting corresponding VPC of your member clusters, then click `Create peering connection`
+button. If two member clusters are in the same account, you can go back to `Peering connections`
+and click `Accept request` to accept this peering connection request.
+
+### Route tables
+
+To enable private IPv4 traffic between instances in peered VPCs, you must add a route to the route
+tables associated with the subnets for both instances. The route destination is the CIDR block (or
+portion of the CIDR block) of the peer VPC and the target is the ID of the VPC peering connection.
+Following are basic steps about how to update public route table for a ClusterSet. For more information,
+see [Configure route tables](https://docs.aws.amazon.com/vpc/latest/userguide/VPC_Route_Tables.html)
+in the Amazon VPC User Guide.
+
+Go to `Virtual private cloud` -> `Route tables`, find the route table name as `west-cluster/PublicRouteTable`,
+Go to `Routes` tab, and click `Edit routes`, you should add a new route by click `Add route` and fill
+the `Destination` with `110.14.0.0/16`, and input the `Target` with `Peering connection`, it will prompt
+the new created peering connection `west-and-east`, choose it as the `Target`. Then click `Save changes`.
+
+Repeat the same steps above with correct settings for another member cluster's route table `east-cluster/PublicRouteTable`.
+
+### Security groups
+
+By default, Antrea Multi-cluster Gateway will use `InternalIP` of EKS Node as serving IP, here we use the
+Gateway IP in the sample table to configure security groups. If you are going to use `ExternalIP` of EKS
+Node as Gateway's serving IP, you should provide the correct IP in security groups.
+
+Please check following steps to configure security groups to allow mutual access by `InternalIP` or
+`ExternalIP` for all member clusters' Gateways.
+
+Go to `Security` -> `Security groups`, search your member cluster's security group with given name,
```

Comment:
Again, I feel no need to talk about a particular way of AWS configuration. Let us just talk about what rule needs to be added, and you can point to the AWS doc about security group if you want. Maybe:

"AWS security groups may need to be configured to allow tunnel traffic to Multi-cluster Gateways, especially when the member clusters are in different VPCs. EKS already creates a security group for each cluster, which should have a description like "EKS created security group applied to ENI that is attached to EKS Control Plane master nodes, as well as any managed workloads.". You can add a new rule to the security group for Gateway traffic. For `west-cluster`, add an inbound rule with source to be `east-cluster`'s Gateway IP `110.14.18.50/32`..."

Reply:
Then we can talk about InternalIP and ExternalIP, like "By default, Multi-cluster Gateway IP should be the `InternalIP` of the Gateway Node, but you may configure Antrea Multi-cluster to use the Node `ExternalIP`. Please use the right Node IP address as the Gateway IP in the security group rule."